src/homework-4/middlewares/errorHandler.ts

```diff
@@ -0,0 +1,26 @@
+import { Request, Response } from "express";
```

Comment:
How about using `winston` package?

Reply:
Sorry for the late answer, I updated it!